romancal/saturation/saturation.py

```diff
@@ -46,9 +46,9 @@ def flag_saturation(input_model, ref_model):
     sat_dq = ref_model.dq.copy()
 
     # Obtain dq arrays updated for saturation
-    gdq_new, pdq_new = flag_saturated_pixels(data, gdq, pdq, sat_thresh,
-                                             sat_dq, ATOD_LIMIT, dqflags.pixel,
-                                             n_pix_grow_sat=0)
```

Comment:
Please add a comment here explaining why this is necessary, e.g. something like `the third returned value is the zero frame which is not used by romancal and is always None`.

Reply:
I added this comment.